From 6590bd26e89c144d0fc5623ee6c4eda64e2de094 Mon Sep 17 00:00:00 2001
From: Steffen Sledz <sledz@dresearch-fe.de>
Date: Fri, 19 Jul 2013 08:14:00 +0200
Subject: [PATCH] systemd-reboot.service: always poweroff on reboot

reboot without poweroff makes no sense on our hipos hardware.

Signed-off-by: Steffen Sledz <sledz@dresearch-fe.de>
---
 units/systemd-reboot.service.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/units/systemd-reboot.service.in b/units/systemd-reboot.service.in
index d99bd3e..547e896 100644
--- a/units/systemd-reboot.service.in
+++ b/units/systemd-reboot.service.in
@@ -14,4 +14,4 @@ After=shutdown.target umount.target final.target
 
 [Service]
 Type=oneshot
-ExecStart=@SYSTEMCTL@ --force reboot
+ExecStart=@SYSTEMCTL@ --force poweroff
-- 
1.8.1.4

